samples/openthread/cli/boards/nrf5340dk_nrf5340_cpuapp_ns.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable Trusted Firmware M
+CONFIG_BUILD_WITH_TFM=y
+
+# Disable FPU since FP_HARD_ABI cannot be used with TF-M, and softfp-float
+# cryptographic libraries are unavailable.
+CONFIG_FPU=n
```

Comment:
Maybe we should discuss some possible issues. This requires another set of pre-built libraries. We would like to switch to sources certification in 1.8.0 but the final decision is not on Nordic. I'm afraid this may lead to some late issues discovery.
Also will this be covered by testing?

Reply:
I just talked about this with @SebastianBoe. Disabling FPU is a temporary workaround and it won't be needed eventually.

Reply:
See https://projecttools.nordicsemi.no/jira/browse/NCSDK-11600

Reply:
We don't need to disable FPU anymore.